### PR TITLE
Add domain to current-federal-non-dotgov.csv

### PR DESCRIPTION
**Pull request by jeffkause**
#  #

## 🗣 Description ##

DOT has requested that we add `mcia4rail.com` so that it shows up in their Trustymail and HTTPS reports. 

## 🧪 Testing ##

I verified it is not currently being picked up in the sources gathered by double checking for its presence in their report.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

File: current-federal-non-dotgov.csv [hunk 1]
```diff
@@ -2063,3 +2063,4 @@ zipstation.net,Federal Agency - Executive,United States Postal Service,,Washingt
 zip-station.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 zipstation.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 zipstation.us,Federal Agency - Executive,United States Postal Service,,Washington,DC,
+mcia4rail.com,Federal Agency – Executive, Department of Transportation,,Washington,DC,
```
Comment (mcdonnnj) [inline current-federal-non-dotgov.csv]: Please adjust this line so that the file remains sorted alphabetically.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
The domain added to the file does not match the domain in the PR description.

**Review by jsf9k — CHANGES_REQUESTED** [review]
Your PR description does not agree with the change that was made.  Please fix whichever one is incorrect.

**Comment by dav3r** [comment]
@jeffkause Is this PR still needed?  If so, please address the comments above.

File: current-federal-non-dotgov.csv (hunk 1 above)
Comment (jeffkause) [reply current-federal-non-dotgov.csv]: Sorry for the typo. The description has been adjusted.

**Comment by jeffkause** [comment]
Yes, the PR is still needed. I fixed the description.

File: current-federal-non-dotgov.csv (hunk 1 above)
Comment (dav3r) [reply current-federal-non-dotgov.csv]: @jeffkause The line you added still needs be moved to maintain the alphabetical ordering of this file.

File: current-federal-non-dotgov.csv [hunk 2]
```diff
@@ -2063,3 +2064,4 @@ zipstation.net,Federal Agency - Executive,United States Postal Service,,Washingt
 zip-station.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 zipstation.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 zipstation.us,Federal Agency - Executive,United States Postal Service,,Washington,DC,
+
```
Comment (dav3r) [inline current-federal-non-dotgov.csv]: Remove blank line to pass linter tests:
```suggestion
```

Comment (dav3r) [reply current-federal-non-dotgov.csv]: Resolved via https://github.com/cisagov/scan-target-data/pull/103/commits/0a8088997fbf5cf9259b375f65472457d086e35f.

**Review by dav3r — APPROVED** [review]
LGTM now, though @jeffkause please rename this PR with a more useful title, e.g. "Add mcia4rail.com to current-federal-non-dotgov.csv"

File: current-federal-non-dotgov.csv (hunk 1 above)
Comment (dav3r) [reply current-federal-non-dotgov.csv]: @mcdonnnj Can this convo now be resolved?

**Review by jsf9k — APPROVED** [review]

File: current-federal-non-dotgov.csv [hunk 3]
```diff
@@ -640,6 +640,7 @@ managepoboxesonline.com,Federal Agency - Executive,United States Postal Service,
 manufacturingusa.com,Federal Agency - Executive,Department of Commerce,,Boulder,CO,
 marketmailselect.com,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 marketmailselect.us,Federal Agency - Executive,United States Postal Service,,Washington,DC,
+mcia4rail.com,Federal Agency – Executive, Department of Transportation,,Washington,DC,
```
Comment (mcdonnnj) [inline current-federal-non-dotgov.csv]:
```suggestion
mcia4rail.com,Federal Agency – Executive,Department of Transportation,,Washington,DC,
```

**Comment by dav3r** [comment]
@jeffkause Has this PR been abandoned?  Can we close it or is it still needed?

**Review by mcdonnnj — APPROVED** [review]

**Review by jsf9k — APPROVED** [review]

**Pull request merged**